src/components/details-view/details-view.js

```diff
@@ -331,6 +331,9 @@ class DetailsView extends Component {
                     id={listSectionId}
                     onToggle={onListToggle}
                     listType={listType}
+                    headerProps={{
+                      role: 'tab'
+                    }}
```

Comment:
If you use one property in object, please write it in one string or use comma, for facilitate support code in future
```suggestion
                    headerProps={{ role: 'tab' }}
```
or
```suggestion
                    headerProps={{
                      role: 'tab',
                    }}
```